src/main/java/com/palantir/gradle/graal/DownloadGraalTask.java

```diff
@@ -33,8 +33,11 @@
 /** Downloads GraalVM binaries. */
 public class DownloadGraalTask extends DefaultTask {
 
-    private static final String ARTIFACT_PATTERN = "[url]/vm-[version]/graalvm-ce-[version]-[os]-[arch].tar.gz";
-    private static final String FILENAME_PATTERN = "graalvm-ce-[version]-[arch].tar.gz";
```

Comment:
I'm not sure that we need to change the local filename. Supportive of adding the OS to match the URL, but I don't think this needs to differ based on the requested version.